frontends/bmd/src/app_root.tsx

```diff
@@ -535,8 +538,98 @@ export function AppRoot({
     /* istanbul ignore else */
     if (newElectionDefinition) {
       updateElectionDefinition(newElectionDefinition);
+
+      void (async () => {
```

Comment:
this isn't the most resilient approach, as it assumes a lot about what the subsequent strings are. But, for a first pass, it has the benefit of being unlikely to fail in a way that breaks the app. If it fails to cache, worst case is that the utterances are delayed on first use.

Reply:
putting this here in the app_root also means that loading the election is not in any way delayed. The utterances are generated in the background, even if the election manager card is removed.

Reply:
So if we miss a string here the failure mode is it just takes a while to be said?